fivetran/resource_connector.go

```diff
@@ -705,6 +706,10 @@ func resourceConnectorUpdateCustomConfig(d *schema.ResourceData) *map[string]int
 		configMap["connection_method"] = v
 	}
 
+	if v, ok := c["is_single_table_mode"].(string); ok && v != "" {
+		configMap["is_single_table_mode"] = v
```

Comment:
```suggestion
		configMap["is_single_table_mode"] = strToBool(v)
```
API expects boolean value. But in provider we use string for any type of values for now.
We should apply type transformation here.